deploy/ui/form.json

```diff
@@ -1548,6 +1504,50 @@
                     }
                   ]
                 },
+                {
+                  "label": "Enable Persistent Storage for kie and maven repositories",
+                  "type": "checkbox",
+                  "required": false,
+                  "jsonPath": "$.spec.objects.servers[*].persistRepos",
+                  "originalJsonPath": "$.spec.objects.servers[*].persistRepos",
+                  "description": "Persist the Maven and KIE repositories on ~/.m2/repository and ~/.kie/repository respectively.",
+                  "fields": [
+                    {
+                      "label": "Enable Persistent Storage for kie and maven repositories",
+                      "type": "fieldGroup",
+                      "jsonPath": "$.spec.objects.servers[*]",
+                      "originalJsonPath": "$.spec.objects.servers[*]",
+                      "displayWhen": "true",
+                      "visible": false,
+                      "fields": [
+                        {
+                          "label": "StorageClass Name",
+                          "required": false,
+                          "jsonPath": "$.spec.objects.servers[*].storageClassName",
+                          "originalJsonPath": "$.spec.objects.servers[*].storageClassName",
+                          "type": "text",
+                          "description": "The storageClassName to use for KIE Server pvc's."
```

Comment:
@spolti This was added in PR #671 but it wasn't added to KieApp.crd.yaml in that PR. Can you please double check that this is related to Kie Server persistent storage and do not affect any other configuration.

Reply:
@jakubschwan ok, I see the jira you've opened, thanks, will take a look.